CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Default owners for everything
+* @mendersoftware/sre-dependabot-reviewers
+
+# Helm chart specific ownership
+/mender/ @mendersoftware/sre-dependabot-reviewers
```

Comment:
I would also put server team in here as half of the templates is simply configuring the application.